src/main/java/com/solvd/laba/Exceptions/ToolPermissionDeniedException.java

```diff
@@ -0,0 +1,17 @@
+package com.solvd.laba.Exceptions;
+
+public class ToolPermissionDeniedException extends Exception {
+
+	private static final long serialVersionUID = 1L;
+	
+	public ToolPermissionDeniedException() {
+	}
+	
+	public ToolPermissionDeniedException(String message) {
+		super(message);
+	}
+	
+	public ToolPermissionDeniedException(String message, Throwable cause) {

```

Comment:
This is not custom.

Reply:
If i can replace your exception with Exception and will not see any difference from user perspective -> it is not custom exception.

Reply:
Hi Vladislav, I tried to resolve this on the Threading Branch. Thanks